tower.yml

```diff
@@ -0,0 +1,5 @@
+reports:
+  multiqc_report.html:
+    display: "MultiQC HTML report"
+  samplesheet.csv:
+    display: "Auto-created samplesheet with collated metadata and FASTQ paths"
```

Comment:
any other simple files to add there?

Reply:
yes. i will check what can be added. we don't have `multiqc_report.html` or any `MultiQC ` output. so that needs to be removed.